src/application/communication/GameClient.java

```diff
@@ -151,6 +158,21 @@ public void setSceneRelativeLocation(TranslatedPoint sceneRelativeLocation)
         }
 
     }
+    private void brokerException()
```

Comment:
never name a method Exception, that word is reserved for Exceptions.
name it brokerError or brokerFault or something else